webapp/components/Group/GroupForm.vue

```diff
@@ -200,53 +253,30 @@ export default {
     }
   },
   computed: {
-    submitDisable() {
-      return (
-        this.formData.name === '' ||
-        this.formData.groupType === '' ||
-        // this.formData.about === '' || // not mandatory
-        this.formData.description === '' ||
-        this.formData.actionRadius === '' ||
-        // this.formData.locationName === '' || // not mandatory
-        this.formData.categoryIds.length === 0
-      )
-    },
-    submitDisableEdit() {
-      return (
-        this.formData.name === this.group.name &&
-        this.formData.slug === this.group.slug &&
-        // this.formData.groupType === this.group.groupType && // can not be changed for now
-        this.formData.about === this.group.about &&
-        this.formData.description === this.group.description &&
-        this.formData.actionRadius === this.group.actionRadius &&
-        this.formData.locationName === (this.group.locationName ? this.group.locationName : '') &&
-        this.sameCategories
-      )
-    },
-    sameCategories() {
-      const formDataCategories = this.formData.categoryIds.map((id) => id).sort()
-      const groupDataCategories = this.group.categories.map((category) => category.id).sort()
-      let equal = true
-
-      if (formDataCategories.length !== groupDataCategories.length) return false
-
-      formDataCategories.forEach((id, index) => {
-        equal = equal && id === groupDataCategories[index]
-      })
-      return equal
+    descriptionLength() {
+      return this.$filters.removeHtml(this.formData.description).length
     },
   },
   methods: {
+    changeGroupType(e) {
+      this.formData.groupType = e.target.value
+    },
+    changeActionRadius(e) {
+      this.formData.actionRadius = e.target.value
+    },
+    updateEditorDescription(value) {
+      this.$refs.groupForm.update('description', value)
+    },
     submit() {
-      const { name, about, description, groupType, actionRadius, locationName, categoryIds } =
+      const { name, goal, description, groupType, actionRadius, locationName, categoryIds } =
         this.formData
       const variables = {
         name,
-        about,
+        goal,
```

Comment:
`goal` is called `about` in the database. So this is not working …

Reply:
is changed